recipe/meta.yaml

```diff
@@ -1,47 +1,44 @@
 {% set name = "submitit" %}
-{% set version = "1.2.1" %}
+{% set version = "1.4.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 27078df91115f366dc4982cae51e6f397becbee1e83a44cabdf311672808efd9
+  sha256: d12cbbfc98a8c1777c4f6e87f73f063dafdba15653bca2984223b038d41f8223
 
 build:
   noarch: python
-  number: 0
   script: {{ PYTHON }} -m pip install . -vv
+  number: 0
 
 requirements:
   host:
-    - cloudpickle >=1.2.1
+    - python >=3.6
+    - flit_core >=3.2,<4
     - pip
-    - python
-    - typing_extensions >=3.7.4.2
   run:
     - cloudpickle >=1.2.1
-    - python
+    - python >=3.6
     - typing_extensions >=3.7.4.2
 
 test:
   imports:
     - submitit
-    - submitit.auto
-    - submitit.core
-    - submitit.local
-    - submitit.slurm
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: https://github.com/facebookincubator/submitit
```

Comment:
add this

Reply:
Why delete the other imports?

Reply:
As many things changed in this library, the reason why it didn't upgrade automatically, I rebuild the recipe. I guess I did a mess.